code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1546,6 +1546,18 @@
 	M.update_transform()
 	..()
 
+/datum/reagent/plastic_polymers
+	name = "plastic polymers"
+	id = "plastic_polymers"
+	description = "the petroleum based components of plastic"
+	color = "#f7eded"
+	taste_description = "plastic"
+
+/datum/reagent/plastic_polymers/on_mob_life(mob/living/M)
+	M.adjustToxLoss(1, 0)
+	. = 1
+	..()
+
```

Comment:
It was a bit of a oversight post testing and realizing it was there. I left it in there in the case the recipe changes later (if people want to make glitter) or somehow people manage to cheat to get the reagent. It should be generic toxic anyway but its dead code not worth bothering about.

> Didn't quite realise the action to make the plastic sheet actually consumed it instantly. Oops.